orca-retrofit/src/main/groovy/com/netflix/spinnaker/orca/retrofit/RetrofitConfiguration.groovy

```diff
@@ -34,40 +37,60 @@ import org.springframework.context.annotation.Scope
 import org.springframework.core.Ordered
 import org.springframework.core.annotation.Order
 import retrofit.RestAdapter.LogLevel
-import retrofit.client.Client
 import retrofit.client.OkClient
 
+import java.util.concurrent.TimeUnit
+
 @Configuration
 @CompileStatic
-@Import(OkHttpClientConfiguration)
+@Import(OkHttp3ClientConfiguration)
 @EnableConfigurationProperties
 class RetrofitConfiguration {
 
-   @Value('${okHttpClient.connectionPool.maxIdleConnections:5}')
+   @Value('${okHttp3Client.connectionPool.maxIdleConnections:5}')
    int maxIdleConnections
 
-   @Value('${okHttpClient.connectionPool.keepAliveDurationMs:300000}')
+   @Value('${okHttp3Client.connectionPool.keepAliveDurationMs:300000}')
    int keepAliveDurationMs
 
-   @Value('${okHttpClient.retryOnConnectionFailure:true}')
+   @Value('${okHttp3Client.retryOnConnectionFailure:true}')
    boolean retryOnConnectionFailure
 
-   @Bean(name = ["retrofitClient", "okClient"])
-   @Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
-   OkClient retrofitClient(@Qualifier("okHttpClientConfiguration") OkHttpClientConfiguration okHttpClientConfig) {
+   @Bean(name = ["retrofitClient"])
+   @Scope(ConfigurableBeanFactory.SCOPE_SINGLETON)
+   Ok3Client retrofitClient(OkHttp3ClientConfiguration okHttpClientConfig) {
```

Comment:
since we're naming beans explicitly ... perhaps more consistent to just call this `ok3Client(...)` and the other `okClient(...)` (rather than `retrofitClient` and `oldRetrofitClient`)?